bindings/nodejs/lib/wallet/account.ts

```diff
@@ -783,20 +783,20 @@ export class Account {
     }
 
     /**
-     * Mint native tokens.
-     * @param params The options for minting tokens.
+     * Create a native token.
+     * @param params The options for creating a native token.
      * @param transactionOptions The options to define a `RemainderValueStrategy`
      * or custom inputs.
      * @returns The minting transaction and the token ID.
      */
-    async prepareMintNativeToken(
-        params: MintNativeTokenParams,
+    async prepareCreateNativeToken(
+        params: CreateNativeTokenParams,
         transactionOptions?: TransactionOptions,
     ): Promise<PreparedMintTokenTransaction> {
```

Comment:
Yeah I think we should defo rename this

Reply:
okay ... suggestions?

Reply:
`CreateOrMintNativeTokenTransaction`?

Reply:
because that sounds like a transaction of native tokens, but this is a transaction of the inception of the native token itself.

Reply:
it's more like a `NativeTokenParamsTransaction`

Reply:
can be closed?